crates/modules/process-monitor/src/lib.rs

```diff
@@ -359,7 +357,7 @@ pub mod test_suite {
                 (Some((false, false)), false),
             ] {
                 // load ebpf and clear interest map
-                let mut bpf = load_test_program(PROCESS_MONITOR_PROBE).unwrap();
```

Comment:
do you still need `bpf_common::program::load_test_program` after this change?

Reply:
Yes, `load_ebpf()` uses `load_test_program()`. It's used only in this crate though..I'll inline it.